tests/RobotFramework/tests/azure/azure_telemetry.robot

```diff
@@ -0,0 +1,93 @@
+*** Settings ***
+Documentation       Purpose of this test is to verify that tedge-mapper-az subscribes the topics that are configured as az.topics
+
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Suite Teardown      Custom Teardown
+
+Test Tags           theme:mqtt    theme:az
+
+
+*** Test Cases ***
+
+Publish measurements to te measurement topic
+    ${timestamp}=        Get Unix Timestamp
+    Execute Command    sudo tedge config set az.topics "te/+/+/+/+/m/+"
```

Comment:
Why set them explicitly when this topic is included by default?

Reply:
Yes, this was done when the older tests were there together, after separating them, this is not needed anymore.